Source/WebCore/platform/ios/WebVideoFullscreenControllerAVKit.mm

```diff
@@ -376,13 +375,13 @@ void returnVideoView() final { }
     });
 }
 
-void VideoFullscreenControllerContext::fullscreenMayReturnToInline()
+void VideoFullscreenControllerContext::fullscreenMayReturnToInline(VideoPresentationModel::FullscreenMayReturnToInlineReply&& reply)
 {
     ASSERT(isUIThread());
-    WebThreadRun([protectedThis = Ref { *this }, this] () mutable {
+    ensureOnMainThread([protectedThis = Ref { *this }, this, reply = WTFMove(reply)] () mutable {
```

Comment:
what changed that made it required to be on the web thread before but not anymore ?

Reply:
`ensureOnMainThread()` is a synonym for `WebThreadRun()`; it will run on the WebThread if it exists, or on the main thread if not. The big difference is that `ensureOnMainThread()` takes a C++ lambda, and `WebThreadRun()` takes an Objective-C block. Because the `FullscreenMayReturnToInlineReply` is non-copyable, that would mean I'd have to wrap the block in a `BlockPtr`. It seemed easier just to use `ensureOnMainThread()` instead.